---a/net/minecraft/network/login/INetHandlerLoginServer.java
+++b/net/minecraft/network/login/INetHandlerLoginServer.java
@@ -4,10 +4,10 @@
 import net.minecraft.network.login.client.C00PacketLoginStart;
 import net.minecraft.network.login.client.C01PacketEncryptionResponse;
 
-public interface INetHandlerLoginServer extends INetHandler
-{
+public interface INetHandlerLoginServer extends INetHandler {
+
     void processLoginStart(C00PacketLoginStart packetIn);
 
     void processEncryptionResponse(C01PacketEncryptionResponse packetIn);
-}
+}
